app/models/podcast.rb

```diff
@@ -19,6 +19,10 @@ class Podcast < ActiveRecord::Base
     DateUpdater.last_build_date(self)
   end
 
+  def account_id
+    URI.parse(prx_account_uri || '').path.split('/').last.to_i
```

Comment:
I don't have a better suggestion (and have been doing this myself)... but these always catch my eye.